packages/components/src/Select/docs/controlled.tsx

```diff
@@ -0,0 +1,25 @@
+import { Select, type Key, Section, Header } from "@hopper-ui/components";
+import { useState } from "react";
+
+export default function Example() {
+    const [selectedKey, setSelectedKey] = useState<Key | null>(null);
```

Comment:
[Question] Was it a team decision to choose to explicitly define types as null instead of undefined? 

Usually in teams I worked with, we let Typescript infer. For this case:

 `
 const [selectedKey, setSelectedKey] = useState<Key>();
 `
 
 This will infer Type as `Key | undefined`

Reply:
I just tested to confirm and setting it to null is the only way to "deselect" an option. 
If I define the selectedKey like you said, then I get this error:
`Argument of type 'null' is not assignable to parameter of type 'SetStateAction<Key | undefined>'.`
Which is why I need to define it like this:
`const [selectedKey, setSelectedKey] = useState<Key | null>();`
but removing that extra null between () works.